CI-CD/Bitbucket/bitbucket-with-pipe.yaml

```diff
@@ -0,0 +1,17 @@
+image: atlassian/default-image:latest
```

Comment:
I do not feel we should be recommending this plugin.  Let's take this offline and discuss.